linter_exclusions.yml

```diff
@@ -217,6 +217,9 @@ aks create:
     enable_cluster_autoscaler:
       rule_exclusions:
       - option_length_too_long
+    cluster_autoscaler_profile:
+      rule_exclusions:
+      - option_length_too_long
```

Comment:
Please add one more short abbreviation for your new argument instead of exclude this rule

Reply:
I don't think there's a reasonable short option for this. This exists as this in the CLI extensions and we'd like to keep it this way.

Reply:
a long argument is not user friendly. You can keep this one same with extension. One argument can have several options.

Reply:
sounds reasonable to me, will make the change